control/preimage/src/treasury_commands.rs

```diff
@@ -0,0 +1,245 @@
+            },
+            TreasuryAsset::USDC => Location {
+                parents: 0,
+                interior: Junctions::X2([
+                    Junction::PalletInstance(50),
+                    Junction::GeneralIndex(1337),
+                ]),
+            },
+        }
+    }
+}
+
+struct Spend {
+    name: &'static str,
+    asset: TreasuryAsset,
+    amount: u128,
+    delay: Option<u32>,
+}
+
+pub const fn dot(amount: u128) -> u128 {
+    amount * UNITS
+}
+
+pub const fn usdc(amount: u128) -> u128 {
+    amount * 1_000_000
+}
+
+pub const SPENDS: [Spend; 23] = [
+    Spend {
+        name: "Operational & Development costs",
+        asset: TreasuryAsset::DOT,
+        amount: dot(179611),
+        delay: None,
+    },
+    Spend {
+        name: "Launch reward",
+        asset: TreasuryAsset::DOT,
+        amount: dot(151699),
+        delay: Some(90 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #1",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: None,
+    },
+    Spend {
+        name: "Milestone completion reward #2",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(30 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #3",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(60 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #4",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(90 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #5",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(120 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #6",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(150 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #7",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(180 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #8",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(210 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #9",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(240 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #10",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(270 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #11",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(300 * DAYS),
+    },
+    Spend {
+        name: "Milestone completion reward #12",
+        asset: TreasuryAsset::DOT,
+        amount: dot(12641),
+        delay: Some(330 * DAYS),
+    },
+    Spend {
+        name: "General Incentive reward #1",
```

Comment:
"Long term incentive reward"

Reply:
is just a comment basically, does not go onchain.